Localization analysis doc touch-up

barentine opened the pull request:
Addresses issue couple dead sphinx refs, outdated names.

**Is this a bugfix or an enhancement?**
enhancement

barentine committed:
fix link to hardware cluster doc

barentine committed:
declutter localization analysis notes

barentine committed:
add PYMEClusterOfOne call to numworkers footnote

barentine committed:
simplify --clusterUI example

barentine committed:
fix other cluster setup link, change VisGUI to PYMEVis in ref (not - still links to old visgui page, ref collision)

David-Baddeley commented on docs/LocalisationAnalysis.rst:
```diff
@@ -169,7 +166,7 @@ This will require the optional dependency django (See also :ref:`cluster setup <
    each of the Acquisition and Server processes (ie limiting the number of workers to 6 in our
    8 core case). This can be done by explicitly specifying the number of workers to launch as
-   an argument eg: ``launchWorkers 6``.
+   an argument eg: ``PYMEClusterOfOne -n 6`` or ``launchWorkers 6``.
```
lauchWorkers is no longer supported
```suggestion
   an argument eg: ``PYMEClusterOfOne -n 6``.
```

David-Baddeley commented on docs/LocalisationAnalysis.rst:
```diff
@@ -32,9 +32,6 @@ python through the firewall, which you should accept.
    rather than a higher level analysis directory. We've done a reasonable ammount of testing, but if something doesn't
    work ``launchWorkers`` is still available (for now, python 2.7 only). Please also let us know so we can fix it.
 
-.. note::
```
I think having a link to the cluster install is still useful ...

barentine committed:
Update docs/LocalisationAnalysis.rst

Co-authored-by: David Baddeley <19475296+David-Baddeley@users.noreply.github.com>

barentine committed:
Merge branch 'master' into locandoc

David-Baddeley committed:
Update LocalisationAnalysis.rst to remove obsolete content

Removed outdated information about the availability of the launchWorkers command and testing notes.
